doc/sources/sphinxext/kivy_pygments_theme.py

```diff
@@ -11,12 +11,12 @@ class KivyStyle(Style):
 
     styles = {
         # No corresponding class for the following:
-        #Text:                     "", # class:  ''
+        # Text:                     "", # class:  ''
         Whitespace:                "underline #ffffff",      # class: 'w'
-        Error:                     "#FF0000 border:#FF0000", # class: 'err'
+        Error:                     "#FF0000 border:#FF0000",  # class: 'err'
```

Comment:
Not sure how to handle this. I ignores such files, because there were comment blocks everywhere, although I think they trigger pep8 error anyway (too many space characters or something).